user-guide/Advanced_Modules/Automation_module/Automation_script_actions/Email.md

```diff
@@ -19,6 +19,7 @@ Use this action to send a notification via email.
    > [!NOTE]
    >
    > - If you want to specify multiple indices for one table parameter, use a semicolon “;” as separator.
+   > - If you want to specify multiple parameters for one element, service, or protocol version, assign them all within a single line.
```

Comment:
@tmsrosa, should ";" also be used as a separator here, similar to when specifying multiple indices for a table parameter? (see suggested change below)

Reply:
It should not.
Purpose of the additional info is to avoid that users create multiple lines for the same element/service/protocol.

Here's an example of how it should:
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/31303691/1a832a7f-03e3-4fc9-a597-8473fb4ab963)